integrations-and-supported-tools/pytorch/scripts/helpers.py

```diff
@@ -0,0 +1,46 @@
+import torch
+import numpy as np
+import matplotlib.pyplot as plt
+from neptune.new.types import File
+import torch.nn.functional as F
+
+# Helper functions
+def save_model(model, name ="model.txt"):
```

Comment:
this would actually be just two lines in the main script)

Reply:
I'm saving the model in the more options example only, so I guess it's ok.

Reply:
I think navigating through 2 example files ( more-options.py and helpers.py) doesn't feel right. 
Obviously as you have a project and look at this in your IDE it's fine but here people will often go to the githab code to see it -> then it is tricky to go from one to another. 

Our goal is to explain those options not to write modular code that would be perhaps more useful in a real-life project.